doc/neps/nep-0012-missing-data.rst

```diff
@@ -903,7 +903,7 @@ before it will allow NA-masked arrays to flow through.
 https://docs.scipy.org/doc/numpy/reference/c-api.array.html#NPY_ARRAY_ALLOWNA
 
 Code which does not follow this advice, and instead just calls PyArray_Check() to verify
-its an ndarray and checks some flags, will silently produce incorrect results. This style
+it's an ndarray and checks some flags, will silently produce incorrect results. This style
```

Comment:
Personally, I would prefer spelling the "is" out.  Formal written out text should not normally use `'`?  The usable surprised me, but seems like generally the more typical spelling.

Reply:
Yes, _useable_ seems correct in American English, although you will find tenfold more _usable_ hits in Google. British dictionaries tend to mention only _usable_.

Reply:
Changed `it's` to `it is`.

Reply:
> British dictionaries tend to mention only _usable_.

"England and America are two countries separated by a common language" -- GBS :)